iphone/Classes/TiUIWebView.m

```diff
@@ -488,6 +488,20 @@ -(void)setScrollsToTop_:(id)value
 	[[self scrollview] setScrollsToTop:scrollsToTop];
 }
 
+- (void)setUserAgent_:(id)args
+{
+    ENSURE_TYPE_OR_NIL(args, NSString);
+    NSString *userAgent = [TiUtils stringValue:args];
+    
+    if (!userAgent) {
+        userAgent = [[TiApp app] systemUserAgent];
+    }
+    
+    NSDictionary *dict = [NSDictionary dictionaryWithObjectsAndKeys:userAgent, @"UserAgent", nil];
+    [[NSUserDefaults standardUserDefaults] registerDefaults:dict];
```

Comment:
Hmm, yeah that's correct. So I propose we move userAgent as a property of when setting the URL. So in other words when setting  the URL we can pass in a single URL or a dict with URL and userAgent. The reason behind this is because to set the userAgent we have to reload the request.  Setting the userAgent would look something like this: 

```
NSURL *url = [NSURL URLWithString:@"someAdress.com"];
NSMutableURLRequest *requestObj = [NSMutableURLRequest requestWithURL:url];
[requestObj setValue:@"Foobar/1.0" forHTTPHeaderField:@"User_Agent"];
[webView loadRequest: requestObj]'; 
```